ts/Studio.ts

```diff
@@ -1228,16 +1232,7 @@ export class Studio {
                     const limit = isCargoType(freightType) ? limits[freightType] ?? 0 : 0;
                     const max = String(limit);
                     const options: InputTextOptions = {min: '0', max};
-                    const form = editNumber(this, frame.state.freightAmount, options, setAmount);
-                    if (isCargoType(freightType) &&
-                        typeof limits[freightType] !== 'undefined' &&
-                        frame.state.freightAmount > limit) {
-                        const title = 'Too much freight';
-                        const c = 'table-warning';
-                        addStat('Freight Amount', form, title, c);
-                    } else {
-                        addStat('Freight Amount', form);
-                    }
+                    addStat('Freight Amount', editNumber(this, frame.state.freightAmount, options, setAmount));
```

Comment:
Please ensure that the commits in your branch are based on the same commit that you are uploading, so that your upload does not roll back any other recent changes to this file.

Reply:
My mistake